.circleci/config.yml

```diff
@@ -50,8 +50,10 @@ jobs:
           path: ./reports/e2e-tests.xml
   osx:
     macos:
-      xcode: 12.5.1
+      xcode: 14.3.1
+    resource_class: macos.m1.medium.gen1
     steps:
+      - run: nvm install 20.6.1 && nvm alias default 20.6.1
```

Comment:
I wonder if installing node this way is messing with the `pkg` library that's building the executables